### ISSUE TITLE
Unencrypted btrfs installation shows no volume groups

### ISSUE THREAD

**Issue by GWeck** [QubesOS/qubes-issues#7644]
### Qubes OS release

R4.1+

### Brief summary

Installing Qubes R4.1 and R4.1.1 with manual partitioning, using btrfs without encryption creates a system that works but seems to have no valid LVM structure.

### Steps to reproduce

Install Qubes R4.1 or R4.1.1, selecting manual partitioning, using the following paramters:

- no disk encryption
- filesystem btrfs
- three partitions `/boot`, `/`, and `swap`.

The installation runs normally and creates a usable system.

### Expected behavior

The command `vgscan` should show the volume group of this installation. Commands like `vgs`, `pvs`, and `lvs`  should provide useful output.

### Actual behavior

The command `vgscan` returns the error `No matching physical volumes found`. As is to be expected in this situation, commands like `vgs`, `pvs`, and `lvs` return empty resuts.

**Comment by DemiMarie** [comment]
Is this a bug or by design?  BTRFS duplicates many of the features of LVM and I am not sure why you want to put BTRFS on top of LVM.  There is no reason that LVM _must_ be used.

**Comment by GWeck** [comment]
If `btfrs` is selected as default for new mount points, the device type is still presented as `LVM` as default. If this is not needed for `btrfs`, it might be more sensible, to grey out this selection if the default is `btfrs` , or at least to change the default to `btrfs` in this case.

Or at least it would be helpful to describe `btrfs` use at installation in more detail in the installation documentation. This would help people like me, who lastly have no idea how `LVM` and  `btrfs` work and what are their differences.

**Comment by rustybird** [comment]
Oh good, the `Custom` storage configuration screen is broken again for the unencrypted use case...

- If `Click here to create them automatically.` is clicked (which is normally the best way to create a Btrfs layout), it's not possible to untick encryption on the `/` Btrfs filesystem afterwards

- If the `/` Btrfs filesystem is added manually, it's automatically put on top of LVM with no way to change that to a raw partition (unless I'm missing something)

As a workaround, `Advanced Custom (Blivet-GUI)` storage configuration looks okay to me. In my test installation, I added the `/` Btrfs filesystem as `Device type: Partition`.

**Comment by marmarek** [comment]
> creates a system that works but seems to have no valid LVM structure.

That's okay. With BTRFS, Qubes will use "file-reflink" storage driver and it doesn't require LVM. If you like, you can still create LVM, for example on separate disk, or for some separate set of VMs.

Are you referring to "No thin pool available" section in initial-setup ([this screen](https://openqa.qubes-os.org/tests/44067#step/firstboot/6))?

**Comment by DemiMarie** [comment]
> Oh good, the `Custom` storage configuration screen is broken again for the unencrypted use case...

I believe the reason for this is that _all_ of the developers use encryption, so the unencrypted case gets no testing.  Therefore, I am not surprised it is broken.

**Comment by rustybird** [comment]
> - If the / Btrfs filesystem is added manually, it's automatically put on top of LVM with no way to change that to a raw partition (unless I'm missing something)

AAAAaaaa(...)aahh I just tried it *again* and this time the `Custom` storage layout screen created a completely fine unencrypted Btrfs storage configuration on a normal partition. Two hours ago the same R4.1.1 installer put it on top of LVM! It said so right in the summary of changes!

**Comment by GWeck** [comment]
@marmarek No - this screen is from the second part of the installation, where the disk setup is already completed. I am referring to the screen during disk partition selection in the first part of the setup, where you create the `/` mount point and have to decide which file system is to be used.

Looking at the installed system, e.g. via `blkid` and `fstab`, it seems that `/` was created as a `btrfs` partition without any `LVM` structure, which is good if that is not needed and explains the behavior of the `LVM` commands. It would be very helpful, though, if this was explained somewhere in the documentation. From the current descriptions, I got the impression that `LVM` would exist in `btrfs`, too. So, a more detailed description of the use and maintenance of a `btrfs` system would help file system agnostic people, like me, enormously.

@rustybird The reason why I used manual partitioning was exactly what you described: Automatic partitioning built a structure with `btrfs`, but did not allow me to uncheck encryption.
